app/controllers/application_controller.rb

```diff
@@ -11,6 +11,7 @@ class ApplicationController < ActionController::Base
   around_action :set_locale
   before_action :redirect_suspended_accounts
   before_action :set_variant
+  before_action :update_recently_active
```

Comment:
I'd like to change this trigger to only occur when:

1. The developer sends a message
2. The developer updates their profile (even with no changes)